tools/python_utils/provenance.cwl

```diff
@@ -6,20 +6,20 @@ id: sample_provenance
 baseCommand:
   - provenance.py
 inputs:
-  - id: infile
+  - id: synapseconfig
     type: File
     inputBinding:
       position: 1
-  - id: synapseconfig
-    type: File
+  - id: argurl
+    type: string
     inputBinding:
       position: 2
 outputs:
   - id: provenance_csv
     type: File
     outputBinding:
-      glob: '*csv*'
+      glob: '*csv'
 label: provenance.cwl
 hints:
   - class: DockerRequirement
-    dockerPull: 'wpoehlm/ngstools:pyscript-3a97325'
+    dockerPull: 'wpoehlm/ngstools:pyscript-8f5f7d9'
```

Comment:
I can't find the Dockerfile for the referenced image. What's in it? Could you use a standard python image instead?

Reply:
This image contains all of the python scripts that I've written for this workflow, under docker/python_utils .  It uses a standard python base image

Reply:
There is an alternative to doing this where you just add the script as a listing and run it with baseCommand or arguments, then you can just dockerPull the standard python base image. Alternatively, if you want to create a special image containing all of these, I think you should publish the dockerFile.

Reply:
I think we should decide how we want to handle this for all future cases since scripts like these will be common

Reply:
Agree, we should develop a best practice. Let's discuss further when you're in.